packages/client/test/end-to-end/erc1271-publish-subscribe.test.ts

```diff
@@ -76,3 +76,125 @@ describe('ERC-1271: publish and subscribe', () => {
         }, TIMEOUT)
     })
 })
+
+describe('ERC-1271: subscribe', () => {
+    let publisherWallet: Wallet
+    let subscriberWallet: Wallet
+    let erc1271ContractAddress: EthereumAddress
+
+    beforeAll(async () => {
+        subscriberWallet = fastWallet()
+        publisherWallet = new Wallet(await fetchPrivateKeyWithGas())
+        erc1271ContractAddress = await deployTestERC1271Contract([toEthereumAddress(subscriberWallet.address)])
+    }, TIMEOUT)
+
+    async function createStream(): Promise<StreamID> {
```

Comment:
Maybe could reduce the code duplication in this test file

Reply:
If you have some ideas for this, more than happy to consider